src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h

```diff
@@ -1682,9 +1682,17 @@ struct YarpSensorBridge::Impl
                     = controlBoardRemapperMeasures.remappedJointPermutationMatrix
                       * controlBoardRemapperMeasures.motorVelocitiesUnordered * M_PI / 180;
 
+                // TO BE CHANGED
+                // For the time being instead of reading the motor acceleration we are reading the
+                // estimated friction torques
+
+                // controlBoardRemapperMeasures.motorAccelerations.noalias()
+                //     = controlBoardRemapperMeasures.remappedJointPermutationMatrix
+                //       * controlBoardRemapperMeasures.motorAccelerationsUnordered * M_PI / 180;
+
```

Comment:
Is this a leftover?

Reply:
No, we are writing the friction torques instead of the motor accelerations. I commented out this to avoid doing twice the conversion `* M_PI/ 180.0` which now is not needed. I also added a comment to remember to change this once there will be a different solution to send the friction torques through yarp.

Is it ok like this or do you prefer not to change sensor bridge and apply the inverse conversion where I read the joint friction torques?